WordPress/src/main/java/org/wordpress/android/ui/pages/PagesFragment.kt

```diff
@@ -137,10 +136,6 @@ class PagesFragment : Fragment() {
         viewModel.onSpecificPageRequested(remotePageId)
     }
 
-    private fun onPageEditFinished(pageId: Long, wasPageUpdated: Boolean) {
```

Comment:
I'm not quite sure what the intentions for the single-line `private` methods were. Please let me know if this is okay.